src/components/GamePanel.vue

```diff
@@ -181,4 +181,48 @@ function addPairToSummary(card1: CardData, card2: CardData) {
 #closeButton:hover {
   cursor: pointer;
 }
+.goback-button {
+  display: inline-block;
+  position: absolute;
+  border-radius: 2%;
+  background-color: transparent;
+  border: rgb(0, 0, 0);
+  color: #000000;
```

Comment:
it would be cool if you could merge the goback button (except for the parts that are different) with the almost identical buttons in StartGameScene